### ISSUE TITLE
Support animated Webp

### ISSUE THREAD

**Issue by rus0n** [OpenFlutter/flutter_image_compress#197]
¿It's possible to support animated Webp in the future? or any solution?

**Comment by NavidHosseini** [comment]
any update

**Comment by tonyCruzNagy** [comment]
Nothing for now

**Comment by AlexV525** [comment]
Thanks for the request. After looking at this, we've decided not to add animated WebP support to this plugin:

- The current pipeline is single-frame end-to-end (Android `Bitmap`, iOS/macOS `UIImage`); animation frames are dropped at decode time.
- AOSP has no built-in animated WebP encoder, so Android would need to ship a `libwebp` NDK binary (extra ABIs, size).
- iOS/macOS could go through `SDWebImageWebPCoder`'s animated encoder, but only after refactoring the whole pipeline to accept multi-frame input (frame durations, loop count, per-frame resize/quality).
- Web (`canvas.toBlob`) can't emit animated WebP at all, so it would be unsupported there.

Effectively this would turn a still-image compression plugin into an animated-image transcoder, which is out of scope. If you need this, a dedicated plugin (e.g. built on top of `libwebp` / `SDWebImageWebPCoder`) is the right shape.

Closing as won't-fix; happy to reopen if there's a plan for a compatible design.
